python/README.md

```diff
@@ -1,5 +1,11 @@
 # Python example for DNB Open Banking APIs
 
+## Requirements
+
+* Python 2.7
+* pipenv - pip install pipenv
+* requests - pip install requests
```

Comment:
Installing dependencies globally is not the best idea, because different projects might depend on different versions, and not something we should encourage. This is why we use pipenv.